### PR TITLE
Fix typo in very frequent log output.

### PR DESCRIPTION
**Pull request by i-am-the-slime**

### CODE REVIEW

**Comment by harti2006** [comment]
:+1:

File: src/main/java/org/zalando/stups/tokens/TokenVerifyRunner.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ public void run() {
                 LOG.debug("{} is open, skip check.", mcb.getName());
             }
         } else {
-            LOG.debug("No TokenVerifier configured, skipp check.");
+            LOG.debug("No TokenVerifier configured, skip check.");
```
Comment (hjacobs) [inline src/main/java/org/zalando/stups/tokens/TokenVerifyRunner.java]: shouldn't it be "skipping"?

Comment (i-am-the-slime) [inline src/main/java/org/zalando/stups/tokens/TokenVerifyRunner.java]: Yes, probably.

**Comment by hjacobs** [comment]
:+1:

**Comment by harti2006** [comment]
:+1:

**Pull request merged**